src/modules/site-v2/templates/primary/map.html

```diff
@@ -0,0 +1,252 @@
+
+      <!-- <div class="col-md-4">
+        <div class="panel panel-default">
+          <div class="panel-heading">
+            <img src="{{ ext_asset('img/icons/selected.png') }}" width="20px;" />
+            <strong>Strain Information</strong>
+          </div>
+          <ul class="list-group">
+
+          <li class="list-group-item">
+            <strong><span class="tooltip-item" data-toggle="tooltip"  data-placement="bottom" title="Name assigned to a group of strains that share the same genome-wide haplotype.">
+              Isotype
+            </span></strong> 
+            <div class="isotype pull-right"></div>
+            <div style="clear:both;"></div>
+          </li>
+
+          <li class="list-group-item">
+            <strong><span class="tooltip-item" data-toggle="tooltip"  data-placement="bottom" title="The name of the strain.">
+              Strain
+            </span></strong> 
+            <div class="strain pull-right"></div>
+            <div style="clear:both;"></div>
+          </li>
+
+          <li class="list-group-item">
+            <strong><span class="tooltip-item" data-toggle="tooltip"  data-placement="bottom" title="Name of strain we distribute as a representative of an isotype.">
+              Reference Strain
+            </span></strong> 
+            <div class="isotype_ref_strain pull-right"></div>
+            <div style="clear:both;"></div>
+          </li>
+
+          <li class="list-group-item"><span class="tooltip-item" data-toggle="tooltip"  data-placement="bottom" title="Dataset in which isotype was originally released"><strong>Release</strong> <div class="release pull-right"></div><div style="clear:both;"></div></li>
+          <li class="list-group-item"><strong>Isolation Date</strong> <div class="isolation_date pull-right"></div><div style="clear:both;"></div></li>
+          <li class="list-group-item"><strong>Latitude, Longitude</strong> <div class="latlng pull-right"><div class="comma"></div></div><div style="clear:both;"></div></li>
+          <li class="list-group-item"><strong>Elevation</strong> <div class="elevation pull-right"><div style="clear:both;"></div></li>
+          <li class="list-group-item"><span class='tooltip-item' data-toggle='tooltip' title='Environment type class'><strong>Landscape</strong></span> <div class="landscape pull-right"></div><div style="clear:both;"></div></li>
+          <li class="list-group-item"><span class='tooltip-item' data-toggle='tooltip' title='Material in or on which nematodes were isolated'><strong>Substrate</strong></span> <div class="substrate pull-right"></div><div style="clear:both;"></div></li>
+          <li class="list-group-item"><strong>Sampled By</strong> <div class="sampled_by pull-right"></div><div style="clear:both;"></div></li>
+        </ul>
+      </div>  {# /panel #}
+     </p>
+    </div>{# /col-md-4 #} -->
+
+</div>
+
+
+
+{% block script %}
+
+<script>
+
+// Icons
+{% autoescape off %}
+var icon_norm = L.icon({
```

Comment:
@orzut: i know we are porting [older] existing code, but wanted to keep this in the radar: for new code, we prefer using `let` and `const` and avoid using `var`.